src/v1-router.js

```diff
@@ -47,6 +47,20 @@ v1router.post('/registrations', async (request, response) => {
   }
 });
 
+/**
+ * Calculates the expiry date by taking the current date and adding five years,
+ * then subtracting a single day.
+ * @returns {Date} the calculated expiry date
+ */
+const calculateExpiryDate = () => {
+  // Get the current date.
+  const expiryDate = new Date();
+  // Add 5 years.
+  expiryDate.setFullYear(expiryDate.getFullYear() + 5);
+  // Subtract 1 day and return as expiry date.
+  return expiryDate.setDate(expiryDate.getDate() - 1);
+};
+
```

Comment:
This isn't actually how trap-registration expiry dates are calculated.

The correct code is in [the v1 registration controller](https://github.com/Scottish-Natural-Heritage/trap-registration-api/blob/main/src/controllers/v1/registration.js#L44-L63).

The logic is 'five calendar years', so it's `the_rest_of_this_year + 4 * whole_years`.